packages/react/src/index.tsx

```diff
@@ -66,9 +66,12 @@ const useReactAdapter = (): ReactAdapter => {
  */
 export function useAtom<T>(atom: Atom<T>): T
 export function useAtom<T>(atom: () => Atom<T>): T
-export function useAtom<T, U>(atom: Atom<T> & Usable<U>, tuple: true): [T, U]
-export function useAtom<T, U>(atom: () => Atom<T> & Usable<U>, tuple: true): [T, U]
-export function useAtom<T, U>(maybeAtom: Atom<T> | (() => Atom<T>), tuple?: boolean): [T, U] | T {
+export function useAtom<T, U>(atom: Atom<T> & Actions<U>, withActions: true): [T, U]
+export function useAtom<T, U>(atom: () => Atom<T> & Actions<U>, withActions: true): [T, U]
+export function useAtom<T, U>(
+  maybeAtom: Atom<T> | (() => Atom<T>),
+  withActions?: boolean
+): [T, U] | T {
```

Comment:
What you're saying is definitely reasonable, however `useAtom` is also a shorthand for creating atoms inline. I think it's documented only in this recipe: https://xoid.dev/docs/recipes/creating-react-custom-hooks. 

```js
const [num, { increment, decrement }] = 
    useAtom(() => CounterModel(value), true)
```
Same thing can be done with a few extra steps, but it's not neat as a single line:
```js
const $counter = useSetup(() => CounterModel(value))
const num = useAtom($counter)
const { increment, decrement } = $counter.actions
```

I didn't add this to the README because I was doubtful about if this feature is going to stay. I was still thinking it may be better to reserve the second argument of `useAtom` for an equality checker function. I was also trying to avoid introducing a third hook.

Now I'm thinking that there could be other ways if custom equality checkers are going to be supported in the future. I'm thinking `withActions` is a more common use case than that.

What do you think?

Reply:
Thanks for explanation! I've used that initializer but I didn't consider how it would be a little more involved using its actions without the tuple option. I don't necessarily think `useAtom` itself need an optional equality checker, but I could maybe see a use case for passing an optional equality checker as a third argument to `create`.